python/README.md

```diff
@@ -73,6 +73,10 @@ For more API calls and environment variables, see [USAGE.md](https://github.com/
 
 To see what you can do with **OpenVINO™ integration with TensorFlow**, explore the demos located in the [examples](https://github.com/openvinotoolkit/openvino_tensorflow/tree/master/examples) repository.
 
+## Docker Support
+Dockerfiles for Ubuntu* 18.04, Ubuntu* 20.04, and TensorFlow* Serving are provided which can be used to build runtime Docker* images for **OpenVINO™ integration with TensorFlow** on CPU, GPU, VPU, and VAD-M. 
+For more details see [docker readme](https://github.com/openvinotoolkit/openvino_tensorflow/tree/master/docker/README.md).
+
```

Comment:
Suggest adding links to prebuilt Docker images here also

Reply:
Links added

Reply:
Can we also add TF Serving docker image links here also? Also, can you change "Prebuilt Images" to "Prebuilt Docker Images"?

Reply:
TF Serving images are present in the same links for ubuntu18_runtime and ubuntu20_runtime with the 2.0.0-serving tag. This will be covered in the README in the respective DockerHub pages.